plugins/modules/storagebox_subaccount.py

```diff
@@ -0,0 +1,473 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2025 Victor LEFEBVRE <dev@vic1707.xyz>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+module: storagebox_subaccount
+short_description: Create, update, or delete a subaccount for a storage box
+version_added: 2.4.0
+author:
+  - Victor LEFEBVRE (@vic1707)
+description:
+  - Create, update, or delete a subaccount for a storage box.
+extends_documentation_fragment:
+  - community.hrobot.robot
+  - community.hrobot.attributes
+  - community.hrobot.attributes.actiongroup_robot
+
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+  idempotent:
+    support: full
+
+options:
+  storagebox_id:
+    description:
+      - The ID of the storage box to query.
+    type: int
+    required: true
+  password_mode:
+    description:
+      - Controls how password updates are handled.
+      - If C(update-if-provided), the password is updated only if provided (default).
+      - If C(ignore-if-exists), password is only used during creation.
+      - If C(set-to-random), password is reset to a randomly generated one.
+    type: str
+    choices: [update-if-provided, ignore-if-exists, set-to-random]
+    default: update-if-provided
+    required: false
+  subaccount:
+    description:
+      - The subaccount configuration.
+    type: dict
+    required: true
+    suboptions:
+      state:
+        description:
+          - Desired state of this subaccount.
+        choices: [present, absent]
+        default: present
+        type: str
+
+      username:
+        description:
+          - Username of the subaccount.
+          - Required for updating or deleting a subaccount (C(state=absent) or update with C(state=present)).
```

Comment:
Generally the syntax is `O(option=value)` or `O(option)`, and `C(value)` or `V(value)`.
```suggestion
          - Required for updating or deleting a subaccount (O(state=absent) or update with C(state=present)).
```